src/create.js

```diff
@@ -11,10 +11,17 @@ const factory = SubmissionError => ({
       return formSubmitSaga(SubmissionError);
     }
     return function* formSubmitSagaComposed () {
-      yield [
-        spawn(root),
-        spawn(formSubmitSaga(SubmissionError))
-      ];
+      if (!all) {
+        yield all([

```

Comment:
I haven't tried this, but looking at it, it looks like this would fail since this is the branch where `all` is *not* defined.